
# [CVE-2022-26280](https://github.com/libarchive/libarchive/issues/1672)

## Description

- `Libarchive v3.6.0 was discovered to contain an out-of-bounds read via the component zipx_lzma_alone_init.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 5.8

## Scores

- **Exploitability Score**:
  - 8.6
- **Impact Score**:
  - 4.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2022-03-28 22:15:09
- **Vulnerability Status**:
  - Modified

## References

- **cve@mitre.org**: https://github.com/libarchive/libarchive/issues/1672
- **cve@mitre.org**: https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/SBYGJICQ7FKDZ2IIOAH423IHWQ6MNONQ/
- **cve@mitre.org**: https://security.gentoo.org/glsa/202208-26
